### ISSUE TITLE
Windows/Mac Installer and Easy Start 

### ISSUE THREAD

**Issue by krolco** [openhardwarecoza/LaserWeb3#135]
Windows / Mac installer
Easy installer for Windows and OSX.

Easy Windows / Mac program execution
Easy way to start LW3 within Windows and OSX.

**Comment by krolco** [comment]
See also #108.

From that thread:
 @funinthefalls:

Here is an easy short term solution to having an easy way to start LaserWeb within Windows until Peter can create a permanent elegant solution.

Enter the following text into Notepad

cd c:\laserweb3 (change this line to whatever the install path is)
start node server-smoothie.js
start chrome http://192.168.0.1:8000/

Save as filename "LW.cmd" and file type as "all types"

I save this to my desktop, and when I want to start LaserWeb I just double click the LW.cmd file

For installing under Windows you can now use NPM install. It is documented at:
https://github.com/openhardwarecoza/LaserWeb3/wiki/Install:-Using-NMP-install-to-simplify-the-process

From @openhardwarecoza:

See https://www.npmjs.com/package/browser-serialport

**Comment by copet80** [comment]
What about packaging and delivering the app with Electron? Overkill?

I believe using Electron will also solve the webcam not working on Mac Safari problem, because then the app is forced running on Chromium.

If there are no obvious objections, I think I'll start tackling this with Electron pretty soon.

**Comment by ghost** [comment]
Node serialport and Electron doesnt play too nicely together,  or so I have
been told. That said,  do you feel up to giving it a go for us? (:

**Comment by copet80** [comment]
Sure that'd be worth trying. What'd be the best way to test the serialport? Connect it to Arduino? Or is it better to test with a LaserWeb compatible hardware?

**Comment by ghost** [comment]
You can test an arduino running Grbl (:

**Comment by copet80** [comment]
No problem. I assume this library suffices?
http://blog.protoneer.co.nz/grbl-arduino-library/

**Comment by ghost** [comment]
Yip,  or the precompiled hex uploaded using xloader or avrdude.  Either is
fine.

**Comment by copet80** [comment]
One thing to consider pretty soon is the certificate signing for Mac OSX. The below article is a bit old but should give an indicator on the process involved. 
https://successfulsoftware.net/2012/08/30/how-to-sign-your-mac-os-x-app-for-gatekeeper/

Having said this, have you got any prior experience on this?

**Comment by pabloevaristo** [comment]
looking for a good controller for my chinese 2.5W engraver, with arduino nano and grbl installed.
I have followed the wiki "Install: Using NPM install to simplify the process" (in windows 7 x64) but got a couple of warnings and can´t start the app.
I attached the output from the console.
Superthanks in advance﻿
![laser1](https://cloud.githubusercontent.com/assets/7711710/20668875/6ad2ecf8-b571-11e6-94da-a1012a6e28ed.png)
![laser2](https://cloud.githubusercontent.com/assets/7711710/20668877/6d3ec2d2-b571-11e6-88fc-35378f896205.png)
